### PR TITLE
Update README and utility install steps

### PR DESCRIPTION
**Pull request by Shabirmean**
## What's this?
 - Updates the README to be able to follow step by step from top to bottom
 - Move the existing README to a new file called `README_DETAILED`
- Add Kubernetes manifest files for the services
- Add some utility steps to make the installation guide easier and less manual steps
- Add supporting scripts to unify some of the manual steps

### CODE REVIEW

**Review by mike-ensor — DISMISSED** [review]

File: anthos-baremetal-edge-deployment/inventory/host_vars/edge-1.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+machine_label: "{{ cluster_name }}"
+# Indicates that this is the primary machine in the cluster (only important for provisioning)
+primary_cluster_machine: true
+# Name of the "store" per the ACM repo
+acm_cluster_name: "{{ cluster_name }}"
+# This machine's IP
+control_plane_ip: "10.223.62.231"
+# The other IPs in the cluster
+control_plane_ips: [
+  "10.223.62.231",
```
Comment (mike-ensor) [inline anthos-baremetal-edge-deployment/inventory/host_vars/edge-1.yaml:14]: Are these the network IPs we want to use?

Comment (mike-ensor) [reply anthos-baremetal-edge-deployment/inventory/host_vars/edge-1.yaml:14]: I would suggest we keep with the VXLAN IP CIDR ranges. I went with a 10.200.x.y

**Review by mike-ensor — CHANGES_REQUESTED** [review]

File: anthos-baremetal-edge-deployment/inventory/host_vars/edge-1.yaml (hunk 1 above)
Comment (Shabirmean) [reply anthos-baremetal-edge-deployment/inventory/host_vars/edge-1.yaml]: Done in https://github.com/GoogleCloudPlatform/anthos-edge-usecases/pull/11/commits/d12e8ca4702c0eb4b5172891448b2ac73da9762b

**Review by mike-ensor — APPROVED** [review]

**Comment by mike-ensor** [comment]
LGTM

**Pull request merged**